doc/api/string_decoder.md

```diff
@@ -50,15 +50,20 @@ added: v0.1.99
 
 Creates a new `StringDecoder` instance.
 
-### stringDecoder.end()
+### stringDecoder.end(buffer)
 <!-- YAML
 added: v0.9.3
 -->
 
+* `buffer` {Buffer} A `Buffer` instance.
```

Comment:
Maybe this (and the same parameter for `write()`) could be a bit more descriptive, since the type is already obvious. Perhaps "Bytes to decode" or something similar?